atlasclient/models.py

```diff
@@ -846,3 +846,62 @@ class SearchFulltext(base.QueryableModel):
     relationships = {'entities': Entity,
                      'attributes': AttributeDef,
                      'fullTextResults': FullTextResult}
+
+
+class SearchSavedCollection(base.QueryableModelCollection):
+    """We can also remove this part but just keeping same structure as above"""
+    def load(self, response):
+        for item in response:
```

Comment:
Why you need to add responses in loop here ?

Reply:
Thanks @verdan for the review. I have added loop becuase the response in a list of AtlasUserSavedSearch. Please suggest.

Reply:
I think the model.load will eventually loop to inflate the response, right?

Reply:
It is failing if I don't use loop as mentioned, response itself is array/list not like other bulk response. I saw this is the only one for which response itself is list. For example,  `GET /v2/entity/guid/{guid} ` and `GET /v2/entity/bulk`  response is `AtlasEntitiesWithExtInfo (JSON)`

But for `GET /v2/search/saved` response is `array of AtlasUserSavedSearch (JSON)` and for `GET /v2/search/saved/{name}` reponse is `AtlasUserSavedSearch (JSON)`

Please let me know your view.